tools/github_helper/main.go

```diff
@@ -284,12 +281,20 @@ func (h helper) updatePullRequest(pr *github.PullRequest, s *github.CombinedStat
 		state = GH.success
 	}
 
-	if state == GH.failure && (h.Codecov || h.Cla) {
+	if state == GH.failure && (len(h.CheckToSkip) > 0) {
 		privilege := true
 		for _, status := range s.Statuses {
 			if *status.State != GH.success {
-				if !(h.Codecov && strings.HasPrefix(*status.Context, "codecov")) && !(h.Cla && strings.HasPrefix(*status.Context, "cla")) {
-					//Failed, and not eligible for any privilege
+				skip := false
+				for _, check := range h.CheckToSkip {
+					if *status.Context == check {
```

Comment:
Well, I think it depends on what possible contexts we have. Are we going to have some confused cases. But in this situation, I think we may prefer false negative to false positive. If we can skip one check while the program block it, well, it's easy to manually check and pass it. On contrary, we may miss a really failure.